health/health.d/backend.conf

```diff
@@ -1,3 +1,13 @@
+# Alert that backend will be disabled soon
+   alarm: backend_metrics_eol
+      on: netdata.backend_metrics
+   units: boolean
+    calc: $now - $last_collected_t 
+   every: 5s
+    warn: $this > 0
+   delay: down 5m multiplier 1.5 max 1h
+    info: Backend is deprecated and it will be removed in a near future, migrate your configuration to exporting.conf
```

Comment:
My suggestion:
"Backends subsystem is deprecated and will be removed soon, move your [backend] configuration section to exporting.conf"

Reply:
@joelhans please look at it.

Reply:
I could clean this up a bit, but it's not a huge issue.

```
     info: The backends subsystem is deprecated and will be removed soon. Migrate your configuration to exporting.conf.
```

Reply:
Thank you @joelhans !